pygaggle/run/evaluate_kaggle_highlighter.py

```diff
@@ -37,11 +37,11 @@
 
 class KaggleEvaluationOptions(BaseModel):
     dataset: Path
+    index_dir: Path
     method: str
     batch_size: int
     device: str
     split: str
-    do_lower_case: bool
```

Comment:
Why we don't have this option anymore?

Reply:
I think transformers deprecated this, it is automatically decided?

Reply:
Oh, I see.. but how does it know if it is cased vs non-cased when we load a custom checkpoint (i.e., one that is not on model zoo)?

Reply:
I've not properly tested this functionality yet, so for the time being, I say we go revert this? I figured that it infers from the vocabulary file and config that everything is either cased/uncased but I might be wrong and it might not even be there in the config, and only from transformers trained models. We shall look into this properly once deadlines are over?

Reply:
Yeah, I think it is safer if we just revert this change for now